os/arch/arm/src/common/up_checkspace.c

```diff
@@ -45,4 +51,13 @@ bool is_kernel_space(void *addr)
 	return false;
 }
 
+#ifdef CONFIG_SUPPORT_COMMON_BINARY
+bool is_common_library_space(void *addr)
+{
+	if (g_lib_binp && addr >= (void *)g_lib_binp->alloc[ALLOC_TEXT] && addr <= (void *)(g_lib_binp->alloc[ALLOC_TEXT] + g_lib_binp->textsize)) {
```

Comment:
How about making declaration of text section address? We don't need to call pointer operation twice, then we can improve the performance.